terraform/services/api-rds/main.tf

```diff
@@ -66,8 +87,13 @@ resource "aws_vpc_security_group_ingress_rule" "db_access_from_mgmt" {
 # Create database subnet group
 
 resource "aws_db_subnet_group" "subnet_group" {
-  name       = "${local.db_name}-rds-subnet-group"
-  subnet_ids = [data.aws_subnet.private_subnet_a.id, data.aws_subnet.private_subnet_b.id]
+  name = var.app == "bcda" ? "${var.app}-${var.env}-rds-subnets" : "${local.db_name}-rds-subnet-group"
+
+  subnet_ids = data.aws_subnets.db.ids
+
+  tags = {
+    Name = var.app == "bcda" ? "RDS subnet group" : "${local.db_name}-rds-subnet-group"
+  }
```

Comment:
It will generally be easier to extend to dpc if the conditional is `var.app == "ab2d"` since ab2d is the outlier.